packages/ui/src/theme/css/component/countryCodeSelect.scss

```diff
@@ -1,3 +1,3 @@
-.amplify-countrycodeselect {
+.amplify-dialcodeselect {
```

Comment:
we are migrating our styling to use the new `dialcodeselect` class but both classes exist on the countryCodeSelect so any overrides that customers have using the old classname will still be applied.

Reply:
Can we record that we will need to clean up class `amplify-countrycodeselect` in the breaking changes ticket?

Reply:
this has been noted in the breaking changes ticket